docs/topics/native/native-cocoapods-dsl-reference.md

```diff
@@ -118,15 +118,21 @@ kotlin {
 The `pod()` function call adds a CocoaPods dependency to the Pod built from this project. Each dependency requires
 a separate function call.
 
-You can specify the name of a Pod library in the function parameters and additional parameter values, like the `version` and
-`source` of the library, in its configuration block.
+You can specify the name of a Pod library in the function parameters and additional parameter values, like the `version`
+and `source` of the library, in its configuration block.
 
-| **Name**      | **Description**                                                                                                                                                                                                                                                                                                                                             | 
-|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `version`     | The library version. To use the latest version of the library, omit the parameter.                                                                                                                                                                                                                                                                          |
+In the function parameters, you can specify the name of a Pod library and the `linkOnly` option to use pod dependencies
+with dynamic frameworks only for linking without generating C-interop bindings.
```

Comment:
I think it's a bit easier to understand this sentence if we rearrange it to explain the WHAT before the HOW.
```suggestion
To use pod dependencies with dynamic frameworks for linking only, without generating C-interop bindings, specify the name of a Pod library and the `linkOnly` option in the function parameters.
```